tslint.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This reminds me we need to migrate `brs` (and now `@hulu/roca`) to `eslint` now that `tslint` is deprecated.  One day!